.gitignore

```diff
@@ -2,6 +2,7 @@
 db_snapshots
 profiles/*_local/
 *compose.env
+!.github/assets/test_scripts/*compose.env
```

Comment:
If you gave the file a different name, you would not need to do the extra exception here.